### ISSUE TITLE
Site Settings: Consider reference to Settings -> General

### ISSUE THREAD

**Issue by fushar** [Automattic/wp-calypso#97154]
Feedback from pdtkmj-3bn-p2#comment-6000

> within the Settings > Site area it seems very minimal here, only showing the Privacy option and Footer credit with Business plan upsell. May be a bit confusing as to how to change things like Site Name, Description, Language, and other General settings unless you knew to go to the WP-Admin area. Perhaps have a link to view these other settings from the Hosting Menu area Settings.

> Comparing the new Settings > Site to old Settings > General, I no longer see the Site profile section that includes the option to add Site Icon, Site Title, Site Tagline, etc. Not sure if that’s intended behavior. I couldn’t find where those were relocated if this is intended behavior.

Note that Site name, tagline etc are Core features, while Privacy and Footer Credit are dotcom-specific. In production, the location of these features are as follows:

- For sites with Default admin interface:
   - Core and dotcom features live in Settings -> General (Calypso).
- For sites with Classic admin interface:
   - Core features live in Settings -> General (wp-admin native screen).
   - Dotcom features live in Hosting -> Site Settings (Calypso screen).

Regardless, should we consider linking back to Settings -> General?

**Comment by matt-west** [comment]
Yeah, it makes sense to include a banner linking to WP Admin in case users are confused.

I included a mockup in the latest iteration of the multi-site dashboard designs. (p9Jlb4-fpU-p2)

![Image](https://github.com/user-attachments/assets/33ca75c5-f565-4ddb-9f4b-f797d6a0260f)

In situ:

![Image](https://github.com/user-attachments/assets/9406a59f-1910-4dfb-baf9-ff00fb1c3440)

**Comment by matt-west** [comment]
@fushar Do you have any feedback on the design shared above?

This banner should work regardless of which approach we take with the navigation.

**Comment by fushar** [comment]
Yep, I like the banner. It's concise, on point, and contextual. Thank you!

**Comment by matt-west** [comment]
Cool. Moving this to `Up Next` as it’s no longer blocked by design.

**Comment by richtabor** [comment]
> Yeah, it makes sense to include a banner linking to WP Admin in case users are confused.

Is there a reason we shouldn't just put these settings in WP Admin instead of having two settings screens?

**Comment by matt-west** [comment]
> Is there a reason we shouldn't just put these settings in WP Admin instead of having two settings screens?

Our goal is to create a separation between the settings related to the site’s hosting and WordPress’ core settings.

In this case, the site title, icon, etc. are core WordPress settings, but we want to account for the case where someone might land in hosting settings looking for them.

**Comment by richtabor** [comment]
> In this case, the site title, icon, etc. are core WordPress settings, but we want to account for the case where someone might land in hosting settings looking for them.

I lean towards keeping hosting-level settings here in Calypso, and site-level settings at the site level. 

I think we need to be diligent in avoiding the confusion between WP Admin and Calypso as we work on untangling them. Where is the line here? There previously wasn't a well-defined line, which is partly why there are many odd, loose, ends that we're untangling. 

I would potentially be in support of a link to edit site settings, rather than include any here.

**Comment by matt-west** [comment]
@richtabor What are you referring to when you mention `Calypso` in this context?

The line we’re trying to draw is between the site management dashboard in WordPress.com and WP Admin for individual sites.

The WP Admin experience should be as close to core WordPress as possible.

Anything that we're adding on top of that (deployment, hosting settings, performance) should either live in the site management dashboard on WordPress.com or is made visible in WP Admin through Jetpack.

**Comment by taipeicoder** [comment]
@crisbusquets it was pending design input. I'll move it to backlog so that we can work on it once it's reprioritized.

**Comment by crisbusquets** [comment]
Thanks, @taipeicoder.

@matt-west does it make sense to tackle this now? I'm asking because we're already working on the multi-site dashboard, and both issues are related.

**Comment by matt-west** [comment]
Let’s pause this pending the outcome of the Forms & Settings atelier. Dotcom settings will need another update based on that work and we can resolve this at the same time.
